### ISSUE TITLE
Add other elements to layouts

### ISSUE THREAD

**Issue by dvirk** [mick-warehime/sixth_corp#100]
blocked by #99 

Make an Element class which is something that takes up space on the screen. It has a height and width and a blit method. The layouts should organize the Elements within themselves, then blit them. For a vertical layout, the sum of element heights is that layout's height (the width is just the max width). Similarly for the horizontal layout. When everything is blitted on the screen, their absolute heights/widths are scaled to fit the screen.

**Comment by mick-warehime** [comment]
we dont need this we can add the status overlay directly. would be nice to have but not essential
